collector/src/main/java/com/bitmovin/analytics/features/FeatureManager.kt

```diff
@@ -0,0 +1,32 @@
+package com.bitmovin.analytics.features
+
+import android.util.Log
+import com.bitmovin.analytics.data.AdEventData
+import com.bitmovin.analytics.data.EventData
+
+class FeatureManager {
+    companion object {
+        val TAG = FeatureManager::class.java.name
+    }
+    private val features: MutableList<Feature<*>> = mutableListOf()
+
+    fun registerFeature(feature: Feature<*>) {
```

Comment:
this method is not used, and as I understand it is entry point to the whole feature tracking ? how is then code from this PR used ?